MANIFEST.in

```diff
@@ -1 +0,0 @@
-include pylhc_submitter/sixdesk_tools/mask_*
```

Comment:
Don't you have to tell hatch/hatchling to include this file?

Reply:
By default, because the files are tracked by version control (git here) they are included at build time. See: https://hatch.pypa.io/1.12/config/build/#file-selection